AutoMapper.AspNetCore.OData.EFCore/LinqExtensions.cs

```diff
@@ -442,13 +442,19 @@ public static List<string> GetSelects(this SelectExpandQueryOption clause)
 
         private static List<string> GetSelects(this SelectExpandClause clause)
```

Comment:
Better to leave this method as it is then remove the [ExpansionIsValid check](https://github.com/AutoMapper/AutoMapper.Extensions.OData/blob/5ea5f6419aa7535d6bab80b74545af31422beab8/AutoMapper.AspNetCore.OData.EFCore/LinqExtensions.cs#L526).

Reply:
Yes this is much cleaner. I also fixed the Web.Tests.